### PR TITLE
Added dockerignore

### LINKED ISSUE #29: Add .dockerignore
Change Dockerfile form copying specific files to copying everything but what is in .dockerignore file

### PR DESCRIPTION
**Pull request by seker212**
Resolves #29

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+**/*.mp3
+
+# Cache directories
+**/*cache*/
+**/*.pyc
+
+# Visual Studio Code
+.vscode/
+
+# particular files
```
Comment (Niewidzialny84) [inline .dockerignore:28]: Poker folder is also unused

**Review by Niewidzialny84 — CHANGES_REQUESTED** [review]
Legit but poker folder should be also included for now in .dockerignore

File: .dockerignore (hunk 1 above)
Comment (seker212) [reply .dockerignore:28]: Done

**Review by Niewidzialny84 — APPROVED** [review]
Legit

**Pull request merged**